src/stdlib_io_disp.fypp

```diff
@@ -3,55 +3,57 @@
 
 submodule (stdlib_io) stdlib_io_disp
 
-    use, non_intrinsic :: stdlib_strings, only: format_string
+    use stdlib_ascii, only: to_string
+    use stdlib_strings, only: format_string
     implicit none
-    character(len=*), parameter :: fmt_r = '(*(g11.4, 1x))'
-    character(len=*), parameter :: fmt_c = '(*(g23.4, 1x))'
+    character(len=*), parameter :: fmt_r = '(*(g12.4, 1x))'
+    character(len=*), parameter :: fmt_c = '(*(g25.0, 1x))'
```

Comment:
I don't know how to control the formatter here. What I want is to keep the output type with 4 significant digits, but the consequence is that when outputting `-1.e10`, the output result is `-0.1000E+ 11`, here will occupy 11 characters width.

Reply:
This format here is set by default and is not shown to users. Do we allow users to customize the output format? This will increase the complexity of `disp`.